docs/source/cores.rst

```diff
@@ -51,6 +51,55 @@ The following is a pictorial of the verification environment .
 For more information on the [fpgalink]() software, firmware, and
 general design information see [makestuff]().
 
+uart
```

Comment:
you can put just a simply summary in the "cores" and then link to the full (datasheet) in the separate file.